addons/pinniped/post-deploy/pkg/configure/configure_test.go

```diff
@@ -311,6 +372,17 @@ func TestPinniped(t *testing.T) {
 			require.Equal(t, test.wantCertManagerClientActions, fakeCertManagerClient.Actions())
 			require.Equal(t, test.wantSupervisorClientActions, supervisorClientset.Actions())
 			require.Equal(t, test.wantConciergeClientActions, conciergeClientset.Actions())
+
+			ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+			defer cancel()
+
+			if test.wantJWTAuthenticator.Name != "" {
```

Comment:
I don't like the `if` statement here really. I'd rather it just be part of the above inputs & outputs, but 🤷

Reply:
shouldn't this go with the `wantConciergeClientActions`? i.e., isn't this assertion made by `kubetesting.NewRootUpdateAction(jwtAuthenticatorGVR, jwtAuthenticatorWithUUIDAudience)`?